frontend/app/views/fragments/event/image.scala.html

```diff
@@ -1,12 +1,10 @@
-@(event: model.RichEvent.RichEvent, sizes: Option[String] = None, lazyLoad: Boolean = true)
+@(img: model.ResponsiveImageGroup, sizes: Option[String] = None, lazyLoad: Boolean = true)
 
-@for(img <- event.imgOpt) {
-    <img
-        @if(!lazyLoad){src="@img.defaultImage"}
-        @if(lazyLoad){data-srcset="@img.srcset"} else {srcset="@img.srcset"}
-        sizes="@sizes.getOrElse("100vw")"
-        alt="@img.altText"
-        class="responsive-img @if(lazyLoad) { js-lazyload} "
-        itemprop="image"
-    />
-}
+<img
+    @if(!lazyLoad){src="@img.defaultImage"}
+    @if(lazyLoad){data-srcset="@img.srcset"} else {srcset="@img.srcset"}
```

Comment:
Could just express this as:

    @if(!lazyLoad){src="@img.defaultImage" srcset="@img.srcset"} else { data-srcset="@img.srcset"}

right?